Source/DafnyDriver/Commands/DafnyCli.cs

```diff
@@ -32,6 +32,7 @@ public static class DafnyCli {
   private static readonly RootCommand RootCommand = new("The Dafny CLI enables working with Dafny, a verification-aware programming language. Use 'dafny -?' to see help for the previous CLI format.");
 
   private static readonly Uri StandardLibrariesDooUri = new("dllresource://DafnyPipeline/DafnyStandardLibraries.doo");
+  private static readonly Uri StandardLibrariesArithmeticDooUri = new("dllresource://DafnyPipeline/DafnyStandardLibraries-arithmetic.doo");
```

Comment:
I know this was from my own commit :) but I think we should use a more accurate suffix here, since this doo file only contains part of the Arithmetic library. Perhaps `-nla-disabled.doo`?

Reply:
Agreed; would hold off until PR is merged in the interest of time if that's OK with you.

Reply:
Oh that's just fine, since this isn't user-facing